### ISSUE TITLE
Connect-Tesla does not work.

### ISSUE THREAD

**Issue by thezim** [SimonWahlin/MyTesla#9]
I was doing my own implentation based in [this](https://tesla-api.timdorr.com/api-basics/authentication) in PowerShell and C# and was running into not getting a redirect either. I tried your implementation and the same appears to be true. I could very well be doing something wrong too in my code.

<img width="1198" alt="image" src="https://user-images.githubusercontent.com/14023826/128769470-ce3c8921-6716-4d3c-bc47-d5a518a9534f.png">

**Comment by SimonWahlin** [comment]
I've had similar problems. I get a captcha javascript in the response and no redirect header.

Changing the userAgent header sometimes works, that was the reason behind parameterizing that.

Once you get a token, it is valid for 45 days so make sure to store it somewhere safe so you dont have to repeat the login-dance. This is what Export-TeslaContext is all about.

**Comment by thezim** [comment]
Yeah, I saw that the captcha hidden input field in the raw response, figured it was the cuplprit in all my testing.

**Comment by SimonWahlin** [comment]
Have you tried using various "legit" userAgent strings?

**Comment by janegilring** [comment]
Got the same problem here. Are there any other workarounds? Will enabling MFA likely remove the captcha requirement?

Also found some approaches for the captcha challenge here:
https://github.com/timdorr/tesla-api/discussions/390

**Comment by SimonWahlin** [comment]
Enabling MFA does not seem to make a difference. I've had some luck with changing the UserAgent but otherwise I dont have a clue.

**Comment by SimonWahlin** [comment]
I've been doing some trials and it seems like I can download the captcha svg and have the user decode it and enter the code in a prompt.

However, after a few trials the API totally stopped answering when calling it from PowerShell. Curl.exe and browser to same URI seems to be working. Somehow Invoke-Webrequest requests does not look the same as a browser regardless of headers. Did some basic tracing with Fiddler and havent found anything obvious but will keep looking.

Will update when I have a working captcha workflow.
